gym_solo/core/rewards.py

```diff
@@ -102,6 +141,48 @@ def compute(self) -> float:
       float: A real-valued number in [-1, 1], where 1 means perfectly upright 
       whilst -1 means that the robot is literally upside down. 
     """
-    _, quat = p.getBasePositionAndOrientation(self._robot_id)
-    unused_x, y, unused_z = np.array(p.getEulerFromQuaternion(quat))
+    _, quat = self.client.getBasePositionAndOrientation(self._robot_id)
+    unused_x, y, unused_z = np.array(self.client.getEulerFromQuaternion(quat))
     return self._fully_upright * y / self._fully_upright ** 2
+
+
+class HomePositionReward(Reward):
+  """Rewards the robot for being in the home position. Currently, this rewards
+  the robot for being orientated properly and being as tall as possible. This
+  will require some experimentation--if the robot ends up jumping to maximize
+  the reward, this might need to be modified to a more intelligent height 
+  reward.
+  """
+  # Found via experimentation--should be close enough to prevent any 
+  # non-trivial errors
+  _quad_standing_height = 0.3
+  _max_angle = np.pi
+
+  def __init__(self, robot_id: int):
+    """Create a new HomePositionReward.
+
+    Args:
+      robot_id (int): The PyBullet body id of the robot
+    """
+    self._robot_id = robot_id
+
+  def compute(self) -> float:
+    """Compute the HomePositionReward for the current state. 
+
+    Returns:
+      float: A real-valued in [0, 1], where 1 is in the home position.
+    """
+    (unused_x, unused_y, z), quat = self.client.getBasePositionAndOrientation(
+      self._robot_id)
+    theta_x, theta_y, unused_z = np.array(
+      self.client.getEulerFromQuaternion(quat))
+
+    x_reward = self._max_angle - abs(theta_x)
```

Comment:
So this comment also involves the next two lines as well. To make it easy for us to know what is going on here, We should calculate the rewards in the following way

`x_reward = (self._max_angle - abs(theta_x))/self._max_angle`
`y_reward = (self._max_angle - abs(theta_y))/self._max_angle`
`orientation_reward = x_reward + y_reward`

Reply:
Heads up that'll change the domain to be `[-2, 2]`. Ack, just going to change it to 

```
x_reward = (self._max_angle - abs(theta_x))/self._max_angle / 2
y_reward = (self._max_angle - abs(theta_y))/self._max_angle / 2
orientation_reward = x_reward + y_reward
```

instead.

Reply:
Based on the code you have, shouldn't it be the following?
```
x_reward = (self._max_angle - abs(theta_x))/(2 * self._max_angle)
y_reward = (self._max_angle - abs(theta_y))/(2 * self._max_angle)
orientation_reward = x_reward + y_reward
```

Reply:
Isn't that the same due to pemdas? xD

Reply:
Based on the discussion we had, we are sticking to my solution based on the readability